to.etc.domui/src/main/java/to/etc/domui/component/input/CheckboxSetInputBase.java

```diff
@@ -105,13 +107,19 @@ private void renderCheckbox(Div setContainer, @NonNull T lv) throws Exception {
 		boolean disa = isDisabled() || isReadOnly();
 		cb.setReadOnly(disa);
 
-		pair.add((NodeBase) cb);
 		IRenderInto<T> cr = m_actualContentRenderer;
 		if(cr == null)
 			cr = m_actualContentRenderer = calculateContentRenderer(lv);
 		Label span = new Label();
 		span.setForTarget((NodeBase) cb);
-		pair.add(span);
+		if(m_labelBeforeCheckbox) {
+			span.addCssClass("label-first");
```

Comment:
All css classes in DomUI use the ui-prefix combined with the component short code.